SQL/migrate/V044__underwear_updates.sql

```diff
@@ -0,0 +1,10 @@
+--
+-- Adds support for underwear updates in PR #6973.
+-- This collates all underwear data into one column, with an additional column for custom colours.
+-- 
+
+ALTER TABLE `ss13_characters`
+	CHANGE COLUMN `underwear` `all_underwear` VARCHAR(32) NULL DEFAULT NULL COLLATE 'utf8mb4_unicode_ci' AFTER `eyes_colour`,
+	DROP COLUMN `undershirt`,
+	DROP COLUMN `socks`,
+	ADD COLUMN `all_underwear_metadata` VARCHAR(7) NULL DEFAULT NULL COLLATE 'utf8mb4_unicode_ci' AFTER `all_underwear`;

```

Comment:
VARCHAR 7 is absolutely not long enough to hold a list. Use JSON or TEXT.

Reply:
If the data stored in that column is json encoded, then definitely use JSON.
Only if the data is not JSON encoded, use TEXT